### PR TITLE
add flow start count model to track # of runs in a flow start

### PR DESCRIPTION
**Pull request by nicpottier**
So we don't get progressively slower on big starts to figure out if we are done.

### CODE REVIEW

File: temba/flows/migrations/0152_flowstartcount.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  -- Table being cleared, reset all counts
+  IF TG_OP = 'TRUNCATE' THEN
+    TRUNCATE flows_flowruncounts;
+    RETURN NULL;
+  END IF;
+
+  -- FlowRun being added
+  IF TG_OP = 'INSERT' THEN
+     -- Is this a test contact, ignore
+     IF temba_contact_is_test(NEW.contact_id) THEN
```
Comment (ericnewcomer) [inline temba/flows/migrations/0152_flowstartcount.py]: soon!

File: temba/flows/tests.py [hunk 2]
```diff
@@ -6596,6 +6594,13 @@ def test_start_flow_action(self):
         # 10 child flow runs should be active waiting for input
         self.assertEqual(FlowRun.objects.filter(flow=child, is_active=True).count(), 10)
 
+        # check our count
+        self.assertEqual(FlowStartCount.get_count(start), 10)
+
+        # recalculate and try again
+        FlowStartCount.populate_for_start(start)
+        self.assertEqual(FlowStartCount.get_count(start), 10)
```
Comment (ericnewcomer) [inline temba/flows/tests.py]: Maybe test post squash?

File: temba/flows/models.py [hunk 3]
```diff
@@ -5214,14 +5214,50 @@ def start(self):
...
+
+        return sql, (distinct_set.start_id,) * 2
+
+    @classmethod
+    def get_count(cls, start):
+        count = FlowStartCount.objects.filter(start=start).aggregate(count_sum=Sum('count'))['count_sum']
+        return count if count else 0
+
+    @classmethod
+    def populate_for_start(cls, start):
```
Comment (rowanseymour) [inline temba/flows/models.py]: seems like this is just called once in a test so maybe it can live there

Comment (nicpottier) [reply temba/flows/models.py]: Going to leave it as it is the kind of thing that is mighty useful to call from a shell.

**Pull request merged**